src/Player.js

```diff
@@ -1,8 +1,9 @@
 class Player {
-  constructor(id, name) {
+  constructor(name, id) {
     this.id = id;
     this.name = name;
-    this.score;
+    this.score = 0;
+    this.scores = [];
```

Comment:
What is the reason for keeping track of scoring (`score` and `scores`) in two places? What are these two properties tracking?